app/views/posts/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with(model: [@user, @post, @comment], local: true) do |form| %>
+    <div>
+      <%= form.label :text, "Comment" %>
+      <%= form.text_area :text %>
+    </div>
+  
+    <div>
+      <%= form.submit "Create Comment" %>
+    </div>
+  <% end %>
```

Comment:
- [ ] Your `post` form is not being rendered. This is because the `@comment` instance variable is not defined in your `post controller`.I believe you meant this form to create a post but instead, you are trying to create a comment with it. Kindly remove the `@comment` instance variable and ensure the form creates a `post ` and not `comment`. 

![image](https://github.com/Samuelwanza/sema/assets/98453979/adb764f2-cb9c-4320-8b8e-b65dadd38369)

##### I am not able to test the `Comment form`   since it is not possible to create a post